libs/net/services/Helpers/ApiService.cs

```diff
@@ -358,7 +360,21 @@ public async Task<IEnumerable<TNO.API.Areas.Services.Models.Ingest.IngestModel>>
     {
         var url = this.Options.ApiUrl.Append($"services/content/references/{contentReference.Source}?uid={contentReference.Uid}");
         var content = JsonContent.Create(contentReference);
-        return await RetryRequestAsync(async () => await Client.SendAsync<API.Areas.Services.Models.ContentReference.ContentReferenceModel>(url, HttpMethod.Put, headers, content));
+        return await RetryRequestAsync(async () => await Client.SendAsync<API.Areas.Services.Models.ContentReference.ContentReferenceModel>(
+            url, HttpMethod.Put, GetHeaders(headers, contentReference), content));
+    }
+
+    private static HttpRequestHeaders? GetHeaders(HttpRequestHeaders? headers, ContentReferenceModel contentReference)
```

Comment:
What is the purpose of this?  We already have a way to add a configured UserAgent to all our requests to the API?  You helped implement this a few months ago.

Reply:
The purpose of this is to add a few more info to the configured UserAgent (which has been implemented a while ago) for verification, so we could confirm if these data have really been updated at this point or by other potential process later on. Thanks